compatibility_lib/compatibility_lib/test_dependency_highlighter.py

```diff
@@ -164,46 +165,29 @@ def setup_test__get_update_priority(self):
 
         with self.patch_checker, self.patch_store:
             highlighter = dependency_highlighter.DependencyHighlighter()
-        ptemp = ("highlighter._get_update_priority('PACKAGE', "
-                 "{'major':%d, 'minor':%d, 'patch':%d}, "
-                 "{'major':%d, 'minor':%d, 'patch':%d}, "
-                 "timedelta(days=%d))")
-
-        cases = []
-        cases.append((
-            priority(level.LOW_PRIORITY, not_updated),
-            eval(ptemp % ((2,5,0)+(2,6,0)+(5,)))
-        ))
-
-        cases.append((
-            priority(level.HIGH_PRIORITY, six_months),
-            eval(ptemp % ((2,5,0)+(2,6,0)+(200,)))
-        ))
-
-        cases.append((
-            priority(level.HIGH_PRIORITY, three_minor),
-            eval(ptemp % ((2,5,0)+(2,8,0)+(13,)))
-        ))
-
-        cases.append((
-            priority(level.LOW_PRIORITY, not_updated),
-            eval(ptemp % ((2,5,0)+(3,0,0)+(29,)))
-        ))
-
-        cases.append((
-            priority(level.HIGH_PRIORITY, thirty_days),
-            eval(ptemp % ((2,5,0)+(3,0,0)+(50,)))
-        ))
-
-        cases.append((
-            priority(level.HIGH_PRIORITY, major_version),
-            eval(ptemp % ((2,5,0)+(3,0,4)+(1,)))
-        ))
-
-        cases.append((
-            priority(level.HIGH_PRIORITY, major_version),
-            eval(ptemp % ((2,5,0)+(5,0,4)+(1,)))
-        ))
+
+        def dictify(args):
```

Comment:
Can we make this args explicit by passing the major, minor, patch into this method?

Reply:
I let `run()` take in `install_tup` and `latest_tup` so that when you call that function, you can group the major, minor, and patch of the different versions and I think it's more straightforward than creating a dict on the fly. `run((2,5,0), (2,6,0), 5)`

And since they were passed in as tuples, I think it's more elegant to let `dictify()` take in a tuple as the argument otherwise the unpackaging can get messy.

Let me know if you disagree with any of that.